### DISCUSSION TITLE
windows版本最新版本的PAGViewer有很多BUG 希望可以解决

### DISCUSSION THREAD

**Discussion by EddieHxc** [Tencent/libpag#2049] [Q&A]
相同版本号下的PAGViewer导出工具在MAC下可以导出，windows系统会报很多错误或者导出失败

**Comment by zhalice2011** [comment]
你好, 请问你指的是 AE 导出插件的导出 PAG 功能还是 PAGViewer 的 文件 -> 导出 功能
<img width="319" alt="image" src="https://github.com/Tencent/libpag/assets/24289349/4f646da3-e065-4354-b777-c20e87b8417d">

具体是什么报错信息

**Reply by EddieHxc** [reply]
你好 是AE导出插件的导出

**Comment by CodeJhF** [comment]
请具体描述一下您遇到的问题呢，最好再提供您的环境信息、出现问题的项目文件等

**Comment by jinwuwu001** [comment]
用户长期无进一步反馈，暂时关闭，有需要可以再开启
